src/components/EasyMde/index.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { TextareaHTMLAttributes, useEffect, useRef } from 'react'
```

Comment:
is this generic enough to be in components ?

Reply:
Yes. It is a generic pancake markdown editor...